ArduPlane/radio.cpp

```diff
@@ -160,26 +139,29 @@ void Plane::rudder_arm_disarm_check()
 			// not at full right rudder
 			rudder_arm_timer = 0;
 		}
-	} else if ((arming_rudder == AP_Arming::RudderArming::ARMDISARM) && !is_flying()) {
-		// when armed and not flying, full left rudder starts disarming counter
-		if (channel_rudder->get_control_in() < -4000) {
-			uint32_t now = millis();
-
-			if (rudder_arm_timer == 0 ||
-				now - rudder_arm_timer < 3000) {
-				if (rudder_arm_timer == 0) {
-                    rudder_arm_timer = now;
-                }
-			} else {
-				//time to disarm!
-				arming.disarm(AP_Arming::Method::RUDDER);
-				rudder_arm_timer = 0;
-			}
-		} else {
-			// not at full left rudder
-			rudder_arm_timer = 0;
-		}
+        return;
 	}
+
+    // full left rudder starts disarming counter
+    if (channel_rudder->get_control_in() >= -4000) {
+        // not at full left rudder
+        rudder_arm_timer = 0;
+        return;
+    }
+    const uint32_t now = millis();
+
+    if (rudder_arm_timer == 0) {
+        // start counter:
```

Comment:
It's not a counter, and I'm not sure this comment helps, at least move it onto the same line maybe?

Reply:
Pre-existing :-P

I'll rearrange this to be minimal-change.  This comment will stay, then, because it won't be a line which needs changing.

We'll also lose the extra comments in htere ("not at full left rudder") etc.  Oh well.

Reply:
This was done.